domains/aarav.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AaravMehta59",
+           "email": "poojaaarav321@gmail.com",
+           "discord": "1139179959965982800"
+        },
+    
+        "record": {
+            "CNAME": "hellow"
```

Comment:
Invalid CNAME record, check https://www.is-a.dev/docs/domain-structure/ on what records you need to put